app/views/maps/_page_map_infowindow.html.erb

```diff
@@ -0,0 +1,35 @@
+<div class="line-height-sans-1">
+  <div class="margin-top-1">
+    <%
+      station_name = facility.official_station_name
+      facility_id = facility.id
+      practice_list = @practices_list
+      names = []
+      taglines = []
+      short_names = []
+      slugs = []
+      practices_to_display_in_info_window facility_id, practice_list, names, taglines, short_names, slugs
+      @num_practices_adopted = names.count
+    %>
+    <p class="margin-0 padding-x-4">
+      <%= link_to "#{station_name} #{ show_common_name(station_name, facility.common_name)}", va_facility_path(facility), class: 'usa-link font-sans-md text-bold' %>
+    </p>
+    <div class="margin-top-0 text-normal theme-type-scale-sm padding-x-4">
+      <%= @num_practices_adopted %> <% if @short_name.length > 0 %> <%= @short_name %> <% else %> Total <% end %> Innovation<% if @num_practices_adopted != 1 %>s<% end %>
+    </div>
+    <div class="text-bold font-sans-3xs">
+      <% i = 0 %>
+      <% while i < names.length do %>
```

Comment:
With the recommendation above, we would just need to loop through the array of practices.

Reply:
Done